phpunit.xml.dist

```diff
@@ -3,6 +3,7 @@
          colors="true"
          xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
          displayDetailsOnTestsThatTriggerDeprecations="true"
+         displayDetailsOnPhpunitDeprecations="true"
```

Comment:
Is there maybe a `failOnPhpunitDeprecation` setting we could use here?

Reply:
indeed there is one :+1: